### PR TITLE
Fixed some spelling and repeated word mistakes

### PR DESCRIPTION
**Pull request by habiblawal1**

### CODE REVIEW

**Comment by planetf1** [comment]
Hi @habiblawal1 Thanks for the contribution.

You will need to 'signoff' the commit to satisfy the 'DCO' check.

This is usually done by `git commit -s -m '#1234 my commit message`

As you have one commit, you could also do:
`git commit --amend -s`
`git force push`

File: site/docs/education/egeria-dojo/running-egeria/user-interfaces/ecosystem-ui/ecosystem-ui-start.md [hunk 1]
```diff
@@ -2,8 +2,6 @@
 
 With the simple environment we have set up, we can now start exploring the Egeria Ecosystem UI.
 
-In later labs you can refer back to this documentation and try out the new with the new environment.
```
Comment (planetf1) [inline site/docs/education/egeria-dojo/running-egeria/user-interfaces/ecosystem-ui/ecosystem-ui-start.md:5]: Did you find this sentence unhelpful and intended to delete?
Or was it an accidental deletion? I can see there are typos in it to correct.

Comment (habiblawal1) [reply site/docs/education/egeria-dojo/running-egeria/user-interfaces/ecosystem-ui/ecosystem-ui-start.md:5]: Yeah I asked David for his opinion on how to reword the sentence and he recommended to just remove the sentence as it didn't make much sense

**Review by davidradl — APPROVED** [review]

**Comment by habiblawal1** [comment]
Solves the issue #713

**Pull request merged**